### PR TITLE
Article fix

### PR DESCRIPTION
**Pull request by Sayanta66**
> Make sure you go through the [Contributing guide](https://github.com/legesher/legesher/blob/master/CONTRIBUTING.md).

> Please delete options that are not relevant.

## Title

This pull request (PR) is made in reference to: 

## Description

There is a wrong article in the documentation. I fixed that.

> What does this PR do?

This PR will fix the document free of that error.

> Recommendations for how to test this?

## Motivation and Context

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

## Testing

* [x] My submission passes the tests?
  - If you did additional testing, please describe in detail how you tested your changes
* [ ] I added new tests for core changes, as applicable?

### CODE REVIEW

**Comment by Sayanta66** [comment]
Please give this PR the hacktoberfest tag

File: README.md [hunk 1]
```diff
@@ -66,7 +66,7 @@
 
 We are all about building bridges to empower others towards the direction of their lives. With **Legesher**, our bridges allow people to code in their native language. Legesher is designed for individuals where English is a second language and for diverse teams that span across many other barriers already.
 
-**Legesher** is a tool that enables anyone to code in their native language. In order to do that we have several projects within the **Legesher** theme to support such an mission. We built **Legesher** on top of [Atom](atom.io), a hackable text editor, and [Git w/ GitHub](https://github.com), a version control tool, in order to give users as much of a personal invitation to programming as we could.
+**Legesher** is a tool that enables anyone to code in their native language. In order to do that we have several projects within the **Legesher** theme to support such a mission. We built **Legesher** on top of [Atom](atom.io), a hackable text editor, and [Git w/ GitHub](https://github.com), a version control tool, in order to give users as much of a personal invitation to programming as we could.
```
Comment (madiedgar) [inline README.md:69]: Thanks for finding this @Sayanta66. Would love to merge this, but it needs to have a little more significant changes for Hacktoberfest. Can I open an issue with some meaningful changes for you to complete?

**Review by madiedgar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Sayanta66) [reply README.md:69]: Yes please. I can do copy-editing quite well. You can create and assign one to me

Comment (madiedgar) [reply README.md:69]: Awesome, I'll create one and assign it to you. If you'd like I can create multiple issues for you, as the README is the same for most of the repos in this project.

**Comment by Sayanta66** [comment]
Yeah please do....assign me after creation. I will complete the work. Thanks a lot for giving this opportunity.

File: README.md (hunk 1 above)
Comment (Sayanta66) [reply README.md:69]: Yeah please do....assign me after creation. I will complete the work. Thanks a lot for giving this opportunity.

**Comment by madiedgar** [comment]
Hey @Sayanta66 please check #92 for more information. 👍

**Pull request closed without merging**